force_wfmanager/central_pane/tests/test_analysis_model.py

```diff
@@ -34,35 +34,41 @@ def test_change_selection(self):
         self.analysis.add_evaluation_step((3, 4))
         self.analysis.add_evaluation_step((5, 6))
 
-        self.analysis.selected_step_index = 0
-        self.analysis.selected_step_index = 2
-        self.analysis.selected_step_index = None
+        self.analysis.selected_step_indices = [0]
```

Comment:
Maybe we could throw a `self.analysis.selected_step_indices = [0, 1]` in here? I mean there's no specific result to test against, but just to check there's no knock-on effects when selecting multiple points.

Reply:
Could also try `.._indices = [0,3]` and check it raises ValueError